requirements.yaml

```diff
@@ -50,6 +50,11 @@ dependencies:
   # In-cluster DB, not recommended
   # for production
   - name: postgresql
-    condition: postgresql.enabled
```

Comment:
I noticed that this line is causing postgresql to install even when the tag is set to false. removing it makes it work properly. This is evident by checking CI runs to see that postgresql was running in the jobs we did not expect it would be.

Reply:
@dimberman do you see this as well?